tox.ini

```diff
@@ -13,6 +13,7 @@ deps = -r requirements/test.txt
 setenv =
   PYTHONPATH = {toxinidir}{/}install
   JUPYTER_PLATFORM_DIRS = 1
+  PYTHONWARNDEFAULTENCODING = 1
```

Comment:
Any reason for using the env variable, instead of the `-X` option? Because that doesn't work with older Python?

Reply:
> Because that doesn't work with older Python?

Yes. Also, this seems easier than placing the option in every call to `python`.